### ISSUE TITLE
Settings menu opens at wrong position in DOM

### ISSUE THREAD

**Issue by Woebin** [Chocobozzz/PeerTube#7129]
### Describe the current behavior

This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG22) [SC 2.4.3 (A) Focus Order](https://www.w3.org/WAI/WCAG22/Understanding/focus-order).

Users who navigate with keyboards or assistive technologies that work similarly should be able to predict where the focus will end up at each step, which means that it should be logical and align with the visually expected reading order.

The video player's settings menu visually appears next to the cog button when opened, but is placed later in the keyboard focus order (it becomes the last available element inside the video player widget). The image below illustrates the current, faulty focus order.

![The video player widget, with each step in the focus order highlighted and numbered.](https://github.com/user-attachments/assets/b933752b-18a8-4b63-943b-358a41dcfd55)

### Steps to reproduce

1. Tab through the interface with the keyboard.
2. Activate the cog button to open the menu.
3. Tab further, notice that the menu isn't immediately after the cog button in the focus order.

### Describe the expected behavior

The menu should be immediately after the button that opens it in the focus order.

### Additional information

* PeerTube instance:
  * URL: https://peertube-eu.howlround.com/
  * Version: 7.2.1
  * NodeJS version: Unknown
  * Ffmpeg version: Unknown

* Browser name, version and platforms on which you could reproduce the bug: MacOS 15.5, Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).

**Comment by Chocobozzz** [comment]
Hi,

~~The settings menu is placed after the control bar, but I'm surprised because when you open it through the cog button it should jump the focus inside the settings menu.~~ reproduced

If I compare it to the youtube player, I notice they put the settings menu before the control bar, but the behaviour is almost the same (I mean the settings menu is not placed just after the cog button).

**Comment by Chocobozzz** [comment]
I think other player settings menu accessibility fixes should also address this issue. If it's not the case please don't hesitate to comment. :)
